namespaces/cloud-platform-live-0.k8s.integration.dsd.io/nick-sample-app-dev/resources/ecr.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=master"
+
+  team_name = "my-team-name"
```

Comment:
Can you please adjust `team_name` and `repo_name` to some value that is different from the documentation example?